package.json

```diff
@@ -50,7 +50,7 @@
     "generate-changelog": "^1.0.2",
     "gulp": "^3.9.1",
     "gulp-babel": "^6.1.2",
-    "istanbul-lib-instrument": "1.3.0",
```

Comment:
This dependency was only added as a hack to force a specific version, it can be completely removed now